releasenotes/notes/process-agent-cobra-4be0f08037b8141e.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
```

Comment:
nit - you can drop this header, it's just to help with filling out various sections.

Reply:
Oh alright!